### PR TITLE
Completes lint for meta folder

### PR DESCRIPTION
**Pull request by pablolmiranda**
- [ ] CHANGELOG.md updated
- [ ] Rebased/mergable
- [ ] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

Should the meta/internal/meta.pb.go be part of the repo and not generated as part of the build process?

Part of #4098

### CODE REVIEW

**Comment by corylanou** [comment]
+1

**Comment by otoolep** [comment]
@pablolmiranda -- CLA must be signed before this can be merged.

+1 from me once that is done.

File: meta/store.go [hunk 1]
```diff
@@ -1200,6 +1200,8 @@ func (s *Store) ShardGroupByTimestamp(database, policy string, timestamp time.Ti
 	return
 }
 
+// ShardOwner look up for a specific shard and return the shard group information
```
Comment (otoolep) [inline meta/store.go]: Nit: look up for a -> looks up a

File: meta/config.go [hunk 2]
```diff
@@ -40,6 +40,7 @@ type Config struct {
 	ClusterTracing      bool          `toml:"cluster-tracing"`
 }
 
+// NewConfig build a new configuration with default values.
```
Comment (otoolep) [inline meta/config.go]: Nit: build -> builds

**Comment by pablolmiranda** [comment]
@otoolep CLA signed.

**Comment by otoolep** [comment]
Thanks @pablolmiranda

**Pull request merged**